README.md

```diff
@@ -125,12 +90,13 @@ Minting beacons for `cardano-swaps` is a tightly controlled process. In order to
 2. The minted beacon uses an empty token name.
 3. The beacon is minted to an address protected by the `cardano-swaps` spending script for a particular trading pair.
 4. The beacon is minted to an address with a staking credential (either a pubkey or a script).
-5. The datum of the output containing the beacon must have the proper beacon symbol.
+5. The datum of the output containing the beacon must have the proper `beaconSymbol` field.
```

Comment:
`BeaconSymbol` is the name of that particular datum, not the name of the field like in the previous version. It is the same as `AskDatum`. It should be fully capitalized and should become:

> must have the proper [`BeaconSymbol` datum with the policy id of the beacon minting policy].